aws_lambda/aws_lambda.py

```diff
@@ -663,3 +679,12 @@ def function_exists(cfg, function_name):
             f['FunctionName'] for f in functions_resp.get('Functions', [])
         ])
     return function_name in functions
+
+
+def read_cfg(path_to_config_file, profile_name):
+    cfg = read(path_to_config_file, loader=yaml.load)
+    if profile_name is not None:
```

Comment:
Instead of:
`if profile_name is not None:`

Couldn't you just do:
`if profile_name:`